### PR TITLE
Forc test change

### LINKED ISSUE #286: Change forc test to cargo test in documentation
#### Description
In versions of forc before 0.25.2, 
`forc test` was the command used to build the project and then run the rust tests, this forc feature was deprecated in 0.26.0
[The CI has already been changed to reflect this](https://github.com/FuelLabs/sway-applications/pull/238), however all the applications in this repo still have the `forc test` command listed as the command to test the application in their respective `README.MD`s. This needs to be changed from

Run the tests
   ```bash
   forc test
   ```

to

Run the tests
   ```bash
   forc build
   cargo test
   ```

### PR DESCRIPTION
**Pull request by brandonsurh**
## Type of change

- Documentation

## Changes

The following changes have been made:

- Changed documentation for all instances of
```
forc test
```

to be

```
forc build
cargo test
```

Changes included `README.md`s and `.docs/contributing-book/src/documentation/project-quality/testing.md` (seen below)
```
This file is the one that is called by `forc build && cargo test` and thus it only contains the modules `functions` and `utils` in order to bring them into scope so that the tests can run.
```

## Notes

- There is still one mention of `forc test` in `.github\workflows\ci.yml`.
- Snippet of code:
```
      - name: Run Forc tests
        run: |
          cd ${{ matrix.project }}
          cargo test
```

## Related Issues

Closes #286

### CODE REVIEW

File: .docs/contributing-book/src/documentation/project-quality/testing.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The repository follows the pattern of putting utility functions in `mod.rs` and
 
 ### `harness.rs`
 
-This file is the one that is called by `forc test` and thus it only contains the modules `functions` and `utils` in order to bring them into scope so that the tests can run.
+This file is the one that is called by `forc build && cargo test` and thus it only contains the modules `functions` and `utils` in order to bring them into scope so that the tests can run.
```
Comment (simonr0204) [inline .docs/contributing-book/src/documentation/project-quality/testing.md:39]:
```suggestion
This file is the entry point for the tests, and thus it contains only the modules `functions` and `utils`. This is what is executed when `cargo test` is run.
```

Comment (simonr0204) [inline .docs/contributing-book/src/documentation/project-quality/testing.md:39]:
```suggestion
This file is the entry point for the tests, and thus it contains only the modules `functions` and `utils`. This is what is executed when `cargo test` is run.
```

**Review by simonr0204 — CHANGES_REQUESTED** [review]

**Review by simonr0204 — APPROVED** [review]

**Comment by simonr0204** [comment]
This is strange - the only files that have changed are the `.md` files, but the CI is failing to build one of the projects. I'll take a closer look later.

File: .docs/contributing-book/src/documentation/project-quality/testing.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ The repository follows the pattern of putting utility functions in `mod.rs` and
 
 ### `harness.rs`
 
-This file is the one that is called by `forc test` and thus it only contains the modules `functions` and `utils` in order to bring them into scope so that the tests can run.
+This file is the entry point for the tests, and thus it contains only the modules `functions` and `utils`. This is what is executed when `cargo test` is run.
```
Comment (Braqzen) [inline .docs/contributing-book/src/documentation/project-quality/testing.md:39]:
```suggestion
The `harness` file is the entry point for the tests, and thus it contains the `functions` and `utils` modules. This is what is executed when `cargo test` is run.
```

File: airdrop/README.md [hunk 3]
```diff
@@ -51,22 +51,13 @@ In order to run the tests you will need to build both the simple-asset and airdr
 
 First, make sure that you are in the root of the simple-asset project i.e. `/path/to/airdrop/simple-asset/<you are here>`.
 
-There is one command required to run the tests for simple-asset. This will both build and test the simple-asset project.
+There are two commands required to run the tests for simple-asset. This will build and test the simple-asset project.
```
Comment (Braqzen) [inline airdrop/README.md:54]:
```suggestion
There are two commands required to run the tests for the simple-asset. The first command builds the contracts and the second command runs the tests.
```

Comment (Braqzen) [reply airdrop/README.md:54]: Note that the sentence above this describes the simple asset however that is not where all the tests are. The context above needs to be updated to make sure that all the tests are run.

**Review by Braqzen — CHANGES_REQUESTED** [review]

**Review by Braqzen — APPROVED** [review]

**Pull request merged**